singleapplication_p.cpp

```diff
@@ -503,12 +503,14 @@ void SingleApplicationPrivate::slotDataAvailable( QLocalSocket *dataSocket, quin
     if ( !isFrameComplete( dataSocket ) )
         return;
 
-    Q_EMIT q->receivedMessage( instanceId, dataSocket->readAll() );
+    auto message = dataSocket->readAll();
```

Comment:
We usually use types explicitly:

```suggestion
    const QByteArray message = dataSocket->readAll();
```

Reply:
Sure thing, applied!